adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAMMainSuite.scala

```diff
@@ -0,0 +1,136 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.adam.cli
+
+import java.io.ByteArrayOutputStream
+import com.google.inject.{ AbstractModule, Guice, Injector }
+import net.codingwell.scalaguice.ScalaModule
+import net.codingwell.scalaguice.InjectorExtensions._
+import org.bdgenomics.adam.cli.ADAMMain.defaultCommandGroups
+import org.scalatest.FunSuite
+
+class ADAMMainSuite extends FunSuite {
+
+  test("default command groups is not empty") {
+    assert(!defaultCommandGroups.isEmpty)
+  }
+
+  test("module provides default command groups") {
+    val injector = Guice.createInjector(new ADAMModule())
+    val commandGroups = injector.instance[List[CommandGroup]]
+    assert(commandGroups == defaultCommandGroups)
+  }
+
+  test("inject default command groups when called via main") {
+    val stream = new ByteArrayOutputStream()
+    Console.withOut(stream) {
+      ADAMMain.main(Array())
+    }
+    assert(stream.toString().contains("flatten"))
```

Comment:
Calling the ADAMMain with an empty Array() will cause the help page to output, correct? If so, we should put a comment that effect and change the assert to look for the word "help" or some such.